qiskit/compiler/transpiler.py

```diff
@@ -250,9 +250,8 @@ def callback_func(**kwargs):
         output_name: A list with strings to identify the output circuits. The length of
             the list should be exactly the length of the ``circuits`` parameter.
         unitary_synthesis_method (str): The name of the unitary synthesis
-            method to use. By default 'default' is used, which is the only
-            method included with qiskit. If you have installed any unitary
-            synthesis plugins you can use the name exported by the plugin.
+            method to use. By default ``'default'`` is used. You can see a list of installed
+            plugins with :func:`~unitary_synthesis_plugin_names`.
```

Comment:
Here, and in all places, the Sphinx cross-ref hasn't linked - `~` doesn't modify the search method, it only affects the display, and the default search path is just matching on prefixes of the current module.  Since this function is documented as being within `qiskit.transpiler.passes.synthesis.plugin`, the prefix match fails and so does the link.

The easiest fix is just to swap the leading `~` for a `.`.

Reply:
oops... thanks for the explanation. Done in 774a40d11dbf6889a9e2215e579dc5420bd8b1d5